UnexpectedDirectory
===================

.. currentmodule:: pystow.utils

.. autoexception:: UnexpectedDirectory
